patches/tinyemu/tinyemu/slirp/slirp.c

```diff
@@ -778,7 +811,12 @@ ssize_t slirp_send(struct socket *so, const void *buf, size_t len, int flags)
 		return len;
 	}
 #endif
+
+#ifdef WASI
+	return send(so->s, buf, len, 0);
```

Comment:
Can we clear OOB flag on the caller of slirp_send rather than entirely remove the flags?